capture_service/constants.h

```diff
@@ -30,14 +30,19 @@ inline constexpr char kXrLayerName[] = "XR_APILAYER_dive";
 inline constexpr char kTargetPath[] = "/data/local/tmp";
 inline constexpr char kGfxrTargetPath[] = "/data/data/";
 inline constexpr char kGfxrReconPyPath[] = "third_party/gfxreconstruct/android/scripts/gfxrecon.py";
-inline constexpr char kDeviceCaptureDirectory[] = "/sdcard/Download/";
+inline constexpr char kDeviceCaptureDirectory[] = "/sdcard/Download";
```

Comment:
nit: maybe better to be `kDeviceCapturePath` just to be consistent with other consts in this file?